study.md

```diff
@@ -111,15 +123,16 @@ The array should contain two strings, "drinking beer" and "propane and propane
 accessories".
 
 ```javascript
-// your answer here
+hank.middle-name = "Rutherford"
+hank.hobbies = ["drinking beer", "propane and propane accessories"]
 ```
 
 ## Removing Dictionary Properties in JavaScript
 
 Remove the `middle name` property from `hank`.
 
 ```javascript
-// your answer here
+delete hank.middle-name
```

Comment:
This will attempt to get access the value of `hank.middle`, if it exists, and then subtract the variable `name` from it.